src/kkt.c

```diff
@@ -184,9 +184,9 @@ static PyGetSetDef KKT_getset[] = {
    "Character representing the quality of primal feasibility.\n"
    "'H', high, 'M', medium, 'L', low, or '?' wrong or infeasible."},
 
-  {"de_ae_row", (getter)KKT_de_ae_col, (setter)NULL,
+  {"de_ae_col", (getter)KKT_de_ae_col, (setter)NULL,
```

Comment:
I'm a little hesitant to merge this pull request with this change for two main reasons:
1. this technically breaks backwards compatibility (existing code that works and uses `de_ae_row` will stop working)
2. there are places in the documentation that need to be updated to reflect this change.

Everything else can be merged to master and the version number can be bumped to v0.4.3. I'll start a v0.5.0-develop branch and we can merge this change there.